quaireaux/math/src/aliquot_sum.cairo

```diff
@@ -10,7 +10,10 @@ use quaireaux_math::unsafe_euclidean_div_no_remainder;
 /// # Returns
 /// * `felt252` - The aliquot sum of the input number.
 fn aliquot_sum(number: felt252) -> felt252 {
-    if number == 1 | number == 0 {
+    if number == 0 {
```

Comment:
Done like this because atm, | isn't short circuiting

Reply:
how about using a loop for this ?

Reply:
as we are using alpha.6, we can't atm.
Will def update it once alpha.7 is out